pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -863,6 +866,50 @@ protected BookieAffinityGroupData internalGetBookieAffinityGroup() {
         }
     }
 
+    private void validateLeaderBroker() {
```

Comment:
If we make it async, the result could be indeterministic. If the bundle-broker mapping is not set by the time loadmanager loads the bundle, it will unload the bundle onto wrong broker.

Reply:
yes, it's dependent and we don't have to make it async here for this admin-api.